theme/forms.py

```diff
@@ -308,15 +308,11 @@ class Meta:
 
     def clean_first_name(self):
         data = self.cleaned_data['first_name']
-        if len(data.strip()) == 0:
-            raise forms.ValidationError("First name is a required field.")
-        return data
+        return data.strip()
```

Comment:
I thought first/last name were still required?  Is there a check somewhere else?

Reply:
@sblack-usu No, first/last name is not required, e.g., organization type users and other users who don't want to provide their names for privacy reasons.